app/root.tsx

```diff
@@ -69,7 +69,7 @@ export const loader = (() => {
   });
 }) satisfies LoaderFunction;
 
-const DataContext = ({ children }: PropsWithChildren) => {
```

Comment:
Caused some weird error for `useLoaderData` being `undefined` ([answer](https://github.com/remix-run/remix/discussions/9681))